aesthetic_ants/penguin/enemy.py

```diff
@@ -2,24 +2,50 @@
 
 from .snowball import Snowball
 from .constants import CollisionType
+from .object import Object
 from .object import PhysicalObject
-from .resources import ENEMY_IMAGE
+from .resources import ENEMY_BIG_IMAGE, ENEMY_FAST_IMAGE
 
 
 class Enemy(PhysicalObject):
     collision_type = CollisionType.ENEMY
     score = 5
 
-    def __init__(self):
-        super().__init__(ENEMY_IMAGE)
-
-        self.velocity_x = random.randint(0, 100)
-        self.velocity_y = random.randint(0, 100)
+    def __init__(self, x, y, speed, hearts, enemy_image):
```

Comment:
`__init__` should just have `(x, y)` as its signature, as its subclasses would override the rest.